apps/studio/src/components/tableview/TableTable.vue

```diff
@@ -954,16 +954,22 @@ export default Vue.extend({
     },
     cellEdited(cell) {
       const pkCells = cell.getRow().getCells().filter(c => this.isPrimaryKey(c.getField()))
+      const oldValue = cell.getOldValue()
+      const currentValue = cell.getValue()
+      const isArray = _.isArray(oldValue)
 
       // some number fields were being converted to strings so were triggered the cellEdited event because tabulator probably `===` stuff
       // If the cell value does fall into this, we don't want anything edited.
-      if (cell.getOldValue() == cell.getValue()) {
+      if (oldValue == currentValue) {
+        // here's another one: If you are editing an array, tabulator is turning it into a comma-delimited string so if it was an array, keep it as such
+        if (_.isArray(oldValue) && !_.isArray(currentValue)) {
```

Comment:
I think we should solve this in the `NullableInputEditor`. Shouldn't the editor just return an Array rather than a string?